packages/gamut-labs/src/landingPage/Description.tsx

```diff
@@ -1,27 +1,31 @@
 import { Markdown } from '@codecademy/gamut';
+import { colors } from '@codecademy/gamut-styles';
 import styled from '@emotion/styled';
 import React from 'react';
 
 import { BaseProps } from './types';
 
-const DescriptionContainer = styled.div`
+const DescriptionContainer = styled.div<{ mode?: 'dark' | 'light' }>`
   margin: 1rem 0 0;
   max-width: 38rem;
+  ${({ mode }) => mode === 'dark' && `color: ${colors.beige};`}
```

Comment:
```suggestion
import { variant } from '@codecademy/gamut-styles';
import { HandlerProps } from '@codecademy/gamut-system';

type DescriptionModes = 'dark' | 'light';

const descriptionVariants = variant<'mode', DescriptionModes>({
  prop: 'mode',
  variants: {
     dark: {
        textColor: 'beige'
     },
     light: {
       textColor: 'navy',
     }
  }
})

const DescriptionContainer = styled.div<{ mode?: DescriptionModes }>`
  ${descriptionVariants}
  margin-top: 1rem;
  max-width: 38rem;
`
```

This is a bit more verbose but it will allow you to drop it anywhere.